src/features/courses/course-route.ts

```diff
@@ -275,6 +303,72 @@ router.put('/topic/:id',
         }
     }));
 
+router.get('/assessment/topic/:id/start',
```

Comment:
This route has too much business logic

Reply:
handled - moved all the checks to a function that returns boolean + message